Formula/flashrom.rb

```diff
@@ -23,6 +23,11 @@ class Flashrom < Formula
   depends_on "libftdi0"
   depends_on "libusb-compat"
 
+  patch do
```

Comment:
Can we add a link to your [PR](https://github.com/flashrom/flashrom/pull/212) here? We probably also want to wait for feedback from upstream before merging this.

Reply:
Yes I missed linking the or it needs to be a comment to merge

Reply:
Done.